### PR TITLE
DDF-2076: Rename RNG methods

### PR DESCRIPTION
**Pull request by tbatie**
#### What does this PR do?
Renamed RandomNumberGenerator methods
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@coyotesqrl 
@bcwaters 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@shaundmorris
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by pklinef** [comment]
:+1:

File: platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/RandomNumberGenerator.java [hunk 1]
```diff
@@ -22,11 +22,11 @@
 
 public class RandomNumberGenerator {
```
Comment (bradh) [inline platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/RandomNumberGenerator.java]: Is this public API? If so, the old name should be deprecated, not removed.

Comment (stustison) [inline platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/RandomNumberGenerator.java]: It was just added in another PR a few days ago. We also only do that if a public API has been released.

Comment (shaundmorris) [inline platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/RandomNumberGenerator.java]: @bradh that is a good question/point. I don't believe this is public API, but somebody can correct me if I'm wrong.

Comment (stustison) [inline platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/RandomNumberGenerator.java]: I don't believe that this qualifies as a public API. This is a utility class that can be embedded. Developers are free to embed the older version if they choose.

**Pull request merged**